Google.PowerShell/Sql/GcSqlInstanceCmdlets.cs

```diff
@@ -153,6 +152,7 @@ public class AddGcSqlInstanceCmdlet : GcSqlCmdlet
         protected override void ProcessRecord()
         {
             InstancesResource.InsertRequest request = Service.Instances.Insert(InstanceConfig, Project);
+            WriteVerbose(string.Format("Adding instance '{0}' to Project '{1}'.", InstanceConfig.Name, Project));
```

Comment:
I think we prefer the less verbose $"Adding instance '{InstanceConfig.Name}' to Project '{Project}'."